test/cpp_extensions/open_registration_extension/torch_openreg/third_party/openreg/csrc/stream.cpp

```diff
@@ -283,8 +283,8 @@ orError_t orDeviceGetStreamPriorityRange(
     return orErrorUnknown;
   }
 
-  // OpenReg have only one priority now.
-  *leastPriority = 0;
+  // OpenReg priority levels are -1 and 0
+  *leastPriority = -1;
```

Comment:
Please refet to the comments located at line 180 of file(test/cpp_extensions/open_registration_extension/torch_openreg/third_party/openreg/csrc/stream.cpp)